visualize_threads.py

```diff
@@ -4,8 +4,30 @@
 import matplotlib.animation as animation
 import sys
 
+
+
+
 fig = plt.figure()
-csv_data = genfromtxt('./data/converted/{0}'.format(sys.argv[1]), delimiter=',')
+
+print("Would you like to visualize the annotated version of this data ? (y/n)")
```

Comment:
so you were trying to read a passed in data.csv, you'd first try to display ./data/annotated/data.csv, and then if that file doesn't exist, automatically fall back on ./data/converted/data.csv

basically try showing annotated version, otherwise try showing just the vanilla converted version, and then catch the exception/print the error message if that doesn't exist either